walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -77,58 +87,68 @@
           Collections.sort(eventTimes, TimeRange.ORDER_BY_START);
           possibleTimes = findTimeRangeGaps(eventTimes, duration);
           if (!possibleTimes.isEmpty()) {
-            // We've found a combination where scheduling is possible, so those are the desired time slot(s).
+            // We've found a combination where scheduling is possible, so those
+            // are the desired time slot(s).
             return possibleTimes;
           }
-          // Re-initialize unavailable time ranges with those of the mandatory attendees.
+          // Re-initialize unavailable time ranges with those of the mandatory
+          // attendees.
           eventTimes = new ArrayList<>(attendeesEventTimes);
         }
       }
-      // Calculate time ranges when there are no optional attendees, or none of the optional attendees can fit the schedule.
+      // Calculate time ranges when there are no optional attendees, or none of
+      // the optional attendees can fit the schedule.
       possibleTimes = findTimeRangeGaps(eventTimes, duration);
     }
     return possibleTimes;
   }
 
   /**
-   * Recursive helper function used to find all combinations of size r of the input array arr[].
-   * @param arr[] The input array.
-   * @param chosen[] Temporary array to store current combination.
-   * @param start Starting index in arr[].
-   * @param end Ending index in arr[].
-   * @param index Current index in chosen[].
+   * Recursive helper function used to find all combinations of size r of the
+   * input array of optional attendees.
+   * @param array The input array of optional attendees that is immutable.
```

Comment:
Rename. This is no longer an array. `inputList`?

Reply:
yup